test/verify/check-services

```diff
@@ -188,6 +189,270 @@ WantedBy=default.target
+        b.click("#boot-or-specific-time .dropdown-menu li[value=2] a")
+        b.wait_in_text("#boot-or-specific-time button", "At specific time")
+        b.set_val("#hr", "24")
+        b.set_val("#min", "6s")
+        b.click("#timer-save-button")
+        # checks for invalid input messages
+        b.wait_text("#servicename-error", "Only alphabets, numbers, : , _ , . , @ , - are allowed.")
+        b.wait_text("#description-error", "This field cannot be empty.")
+        b.wait_text("#command-error", "")
+        b.wait_text("#min-error", "Minute needs to be a number between 0-59")
+        b.wait_text("#hr-error", "Hour needs to be a number between 0-23")
+        b.click("#drop-repeat button")
+        b.click("#drop-repeat .dropdown-menu li[value=10080] a")
+        b.click("[data-content='add'][data-index=0]")
+        b.wait_present("[data-content='minutes'][data-index=1]")
+        b.wait_present("[data-content='hours'][data-index=0]")
+        b.set_val("[data-index=0][data-content='hours']", "s")
+        b.set_val("[data-index=1][data-content='minutes']", "w")
+        b.click("#timer-save-button")
+        b.wait_present("[data-content='hr-error'][data-index=0]")
+        b.wait_text("[data-content='hr-error'][data-index=0]", "Hour needs to be a number between 0-23")
+        b.wait_present("[data-content='min-error'][data-index=1]")
+        b.wait_text("[data-content='min-error'][data-index=1]", "Minute needs to be a number between 0-59")
+        # creates a new yearly timer at 2020-01-01 16:00 and at 2021-01-01 01:22
+        b.set_val("#servicename", "yearly_timer")
+        b.set_val("#description", "Yearly timer")
+        b.click("#drop-repeat button")
+        b.click("#drop-repeat .dropdown-menu li[value=525600] a")
+        b.click("[data-content='add'][data-index=0]")
+        b.wait_present("[data-content='datepicker'][data-index=0]")
+        b.set_val(".bootstrap-datepicker[data-index=0]","2020-01-01")
+        b.set_val("[data-index=0][data-content='hours']", "16")
+        b.set_val("[data-index=0][data-content='minutes']", "00")
+        b.wait_present("[data-content='datepicker'][data-index=1]")
+        b.set_val(".bootstrap-datepicker[data-index=1]","2021-01-01")
+        b.set_val("[data-index=1][data-content='hours']", "01")
+        b.set_val("[data-index=1][data-content='minutes']", "22")
+        b.click("#timer-save-button")
+        b.wait_popdown("timer-dialog")
+        b.wait_present(svc_sel('yearly_timer.timer'))
+
+        m.execute("systemctl start yearly_timer.timer")
+        b.wait_present(svc_sel('yearly_timer.timer'))
+        b.wait_in_text(svc_sel('yearly_timer.timer'), "Yearly timer")
+        self.assertIn("Wed 2020-01-01 16:00", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-01-01 16:10:00'")
+        b.wait_present(svc_sel('yearly_timer.timer'))
+        self.assertIn("Fri 2021-01-01 01:22", m.execute("LC_ALL=C systemctl list-timers"))
```

Comment:
This looks like a race condition ([failed test](http://fedorapeople.org/groups/cockpit/logs/pull-4645-a1023602-verify-centos-7/log.html)) - I think systemd needs some time to update this file. Why not wait for this info to be properly displayed in the cockpit ui? Then you can use the standard wait functionality for the text to be updated.